packages/wallets/src/components/WalletListCardBadge/WalletListCardBadge.tsx

```diff
@@ -14,7 +14,7 @@ const WalletListCardBadge: React.FC<TProps> = ({ isDemo, label }) => {
         'wallets-list-card__badge--demo': isDemo,
     });
 
-    const formattedLabel = label === 'virtual' ? 'Demo' : label?.toUpperCase() || 'SVG';
+    const formattedLabel = label === 'virtual' ? 'Demo' : label?.toUpperCase();
```

Comment:
This is still used by the `DerivAppsTradingAccount` component, so I didn't completely remove it.